README.md

```diff
@@ -211,6 +211,42 @@ Below is a list of available tabs in a GDPR privacy manager:
     }
 ```
 
+# GetSpConsent
+
+Getter for SpConsents data. After calling, checks platform(Android or ios) and return SPConsents structure:
+
+```c#
+    SpConsents
+        |-- gdpr?
+        |   |-- applies: bool
+        |   |-- consents: GdprConsent
+        |       |-- uuid: String?
+        |       |-- tcData: Map<String, String>
+        |       |-- grants: Map<String, GDPRPurposeGrants>
+        |       |-- euconsent: String
+        |       |-- acceptedCategories: List<String>
+        |       |-- webConsentPayload: String
+        |       |-- consentStatus: ConsentStatus
+        |-- ccpa?
+            |-- applies: bool
+            |-- consents: CcpaConsent
+                |-- uuid: String?
+	        |-- rejectedCategories: List<String>
+                |-- rejectedVendors: List<String>
+                |-- status: String?
+                |-- uspstring: String
+                |-- childPmId: String
+                |-- signedLspa: bool
+                |-- webConsentPayload: String
+                |-- consentStatus: ConsentStatus?
+```
+
+This method may return null. Sample usage:
+
+```c#
+    CMP.GetSpConsents()
+```
+        
```

Comment:
A common question we receive is: how do I know the user consented to a given vendor? I suggest adding the following:


```md
## Verifying consent to a given vendor

### IAB Vendors
If the vendor you're interested in verifying consent for is part of the IAB group, you don't need to manually check for consent. Our SDK implements the Transparency Consent Framework (TCF) spec and IAB vendors know how to retrieve consent from the local storage. 

### Non-IAB vendors (aka custom vendors)
For vendors that are not part of the IAB group, you can verify the user consented to it with the following:
     
```c#
     consents = CMP.GetSpConsents();
     
     // for GDPR
     bool isMyGDPRVendorConsented = consents.gdpr.consents.grants["a_vendor_id"].vendorGrant;
     
     // for CCPA
     bool isMyCCPAVendorConsented = consents.ccpa.consents.status != "rejectedAll" ||
                                       consents.ccpa.consents.rejectedVendors.Contains("a_vendor_id");
    ```
```